src/notifications.c

```diff
@@ -1357,6 +1486,10 @@ static int ncntf_event_store(time_t etime, const char* content)
 				if (r == -1) {
                     goto write_failed;
                 }
+				if(ncntf_write_end_marker(s, etime64, MAGIC_END_MARKER) == -1) {
```

Comment:
indentation in this part seems weird, please review it, we were using tabs for indentation in libnetconf